src/int.rs

```diff
@@ -594,6 +599,17 @@ impl<'de> Deserialize<'de> for Int {
     }
 }
 
+#[cfg(feature = "float_deserialize")]
+#[inline(always)]
+pub(crate) fn is_acceptable_float(float: f64) -> bool {
```

Comment:
I could put this into a separate module but I'm not sure what to call that. Or I could put this in `lib.rs`.

Reply:
Moving it to `lib.rs` sounds good.